### ISSUE TITLE
Dynamic mode only works on one monitor.

### ISSUE THREAD

**Issue by RoyalFoxy** [torchgm/RoundedTB#249]
![image](https://user-images.githubusercontent.com/48829979/191005777-f6f7bac2-ab40-4b13-81ee-d1b2045e4f0a.png)
This image shows what I mean

These are my RoundedTB settings:
![image](https://user-images.githubusercontent.com/48829979/191005859-b2f8d270-19a1-4b9d-82c2-0218a79b55af.png)

I have nothing else like TranslucentTB open (closed it to make sure it's not what's causing the issue)

I'm using the version from the Microsoftstore aka v1.3.1.0

I've seen people talk about a autohide feature too, is that something experimental perhaps on a version which could fix my problem too?

**Comment by MaxJW** [comment]
I think your issue may be the same as https://github.com/torchgm/RoundedTB/issues/116 ?

**Comment by MaxJW** [comment]
Yeah unfortunately this is a known problem, see the 'Known Issues' section under the 'Help and About' button in the application. Hopefully it can be fixed soon!

**Comment by Waterfiend** [comment]
Is this fixed yet?

**Comment by nirurin** [comment]
> Is this fixed yet?

Haha I also came here today after finding this app, to ask the same question!

Still a great app, especially tag-teamed with translucentTB on an OLED monitor (auto hide taskbar working fine). Just a shame it's not working on my second monitor.
